index.js

```diff
@@ -3,32 +3,33 @@ const exec = require('child_process').exec
 const fs = require('fs')
 
 try {
-  installSFDX()
+  installSF()
 } catch (error) {
   core.setFailed(error.message)
 }
 
-function installSFDX(){
-  var download = 'wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz -q -P /tmp'
-  var createDir = 'mkdir sfdx'
-  var unzip = 'tar xJf /tmp/sfdx-linux-x64.tar.xz -C sfdx --strip-components 1'
-  var install = 'echo "`pwd`/sfdx/bin" >> $GITHUB_PATH'
-  var version = 'sfdx/bin/sfdx --version && sfdx/bin/sfdx plugins --core'
-  exec(download+' && '+createDir+' && '+unzip+' && '+install+' && '+version, function(error, stdout, stderr){
+function installSF(){
+  var download = 'wget https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-linux-x64.tar.xz -q -P /tmp'
+  var createDir1 = 'mkdir ~/cli'
+  var createDir2 = 'mkdir ~/cli/sf'
+  var unzip = 'tar xJf /tmp/sf-linux-x64.tar.xz -C ~/cli/sf --strip-components 1'
```

Comment:
All you need is `mkdir -p ~/cli/sf`, which creates all non-existing directories too.

Reply:
done